tests/driver_at24cxxx/main.c

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    AT24CXXX test application
+ *
+ * Tested with bluepill and AT24C256
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "xtimer.h"
+
+#include "at24cxxx.h"
+#include "at24cxxx_params.h"
+
+#ifndef AT24CXXX_ERASE
+#define AT24CXXX_ERASE          (0)
+#endif
+
+#define WRITE_BYTE_POSITION     (12U)
+#define WRITE_BYTE_CHARACTER    'A'
+
+#define WRITE_POSITION          (444U)
+#define WRITE_CHARACTERS        { 'B', 'E', 'E', 'R', '4', \
+                                'F', 'R', 'E', 'E', '\0' }
+
+#define SET_POSITION            (900U)
+#define SET_CHARACTER           'G'
+#define SET_LEN                 (20U)
+
+int main(void)
+{
+    puts("Starting tests for module at24cxxx");
+
+    at24cxxx_t at24cxxx_dev;
+    int check;
+
```

Comment:
How about adding something like
```C
    printf("EEPROM size: %d byte\n", AT24CXXX_EEPROM_SIZE);
    printf("Page size  : %d byte\n", AT24CXXX_PAGE_SIZE);
```

That `AT24CDEV ?= at24c256` is a bit of a foot gun, I didn't realize the wrong EEPROM type was selected.

Should `AT24CDEV` be set by the board?

Reply:
For sure I could add these printf()´s.

You mean whether `AT24CDEV` should be set in the Makefile of the board?
No - I guess it is better to do the usual `USEMODULE += at24c256`, instead.
So I´ll remove `AT24CDEV`.

Do you wan´t me to add your patches to the PR?

Reply:
> No - I guess it is better to do the usual USEMODULE += at24c256, instead.

Sounds good.

> Do you want me to add your patches to the PR?

That'd be great!